### PR TITLE
[Trivial, Doc] Describe maxmempool and mempoolminfee in the getmempoolinfo RPC help.

### PR DESCRIPTION
**Pull request by paveljanik**
Add missing documentation to the `getmempoolinfo` RPC call.

What about mexmempool -> max, mempoolminfee -> minfee?

### CODE REVIEW

**Comment by sipa** [comment]
utACK

**Comment by pstratem** [comment]
unACK

**Comment by sawmarku** [comment]
Unmmmm

**Comment by paveljanik** [comment]
@sawmarku ?

File: src/rpcblockchain.cpp [hunk 1]
```diff
@@ -798,6 +798,8 @@ UniValue getmempoolinfo(const UniValue& params, bool fHelp)
             "  \"size\": xxxxx                (numeric) Current tx count\n"
             "  \"bytes\": xxxxx               (numeric) Sum of all tx sizes\n"
             "  \"usage\": xxxxx               (numeric) Total memory usage for the mempool\n"
+            "  \"maxmempool\": xxxxx          (numeric) Maximum memory usage for the mempool\n"
```
Comment (laanwj) [inline src/rpcblockchain.cpp]: Nit: all the other RPC helps have commas after all but the last field in an object, to make it look more like actual JSON:
```
"  \"usage\": xxxxx,
"  \"maxmempool\": xxxxx,
"  \"mempoolminfee\": xxxxx
```
Not that I care deeply about this (or at all, really), but you may preempt someone that wants to make that update later :)

Comment (paveljanik) [inline src/rpcblockchain.cpp]: I can add the commas in the other lines as well (wanted to minimize changes).

Comment (laanwj) [inline src/rpcblockchain.cpp]: Sure, let's do that

**Comment by fanquake** [comment]
utACK

**Pull request merged**